mobile/screens/Home.js

```diff
@@ -1,118 +1,78 @@
-import React, { useState } from 'react';
-import { Text, View, StyleSheet } from 'react-native';
-import { Button, Dialog, Portal, Checkbox } from 'react-native-paper';
-import axios from 'axios';
-import { api } from '../config/Api';
-import Alert from '../components/Alert';
-import { height } from '../config/DeviceDimensions';
-export default function Home({ route }) {
-  const { user } = route.params;
-  console.log('user', user);
-  const [alertVisible, setAlertVisible] = useState(false);
-  const [alertText, setAlertText] = useState('');
-  const [welcomeVisible, setWelcomeVisible] = useState(true);
-  const [taskVisible, setTaskVisible] = useState(false);
-  const [checkedState, setCheckedState] = useState({
-    Work: false,
-    Reading: false,
-    Exercise: false,
-    Break: false,
-  });
-  const handleCheckboxClick = checkboxLabel => {
-    setCheckedState(prevState => ({
-      ...prevState,
-      [checkboxLabel]: !prevState[checkboxLabel],
-    }));
-  };
-  function welcomeToTaskDialog() {
-    setTaskVisible(true);
-    setWelcomeVisible(false);
-  }
-  function timeout(delay) {
-    return new Promise(resolve => setTimeout(resolve, delay));
-  }
-  async function putInitialTaks() {
-    await axios
-      .put(`http://${api}/user/update-preferred-tasks`, {
-        username: user.username,
-        preferredTasks: checkedState,
-      })
-      .then(async r => {
-        console.log(r.data.message);
-        setAlertText(r.data.message);
-        setTaskVisible(false);
+import React, { useEffect } from 'react';
+import { ScrollView, StyleSheet, Image } from 'react-native';
+import { Button, Text, useTheme } from 'react-native-paper';
+import { LinearGradient } from 'expo-linear-gradient';
+import { useSession } from '../context/SessionContext';
+import logo from '../assets/logo.png'; // Ensure the path to your logo is correct
 
-        setAlertVisible(true);
-        await timeout(2000);
-        setAlertVisible(false);
-      })
-      .catch(async error => {
-        console.log('Error', error);
-        setTaskVisible(false);
-        setAlertText('Error updating preferences');
-        setAlertVisible(true);
-        await timeout(1000);
-        setAlertVisible(false);
-      });
-  }
-  function welcome() {
-    return (
-      <Dialog visible={welcomeVisible} onDismiss={() => welcomeToTaskDialog()}>
-        <Dialog.Title>Welcome</Dialog.Title>
```

Comment:
@bdesollar did you move this functionality for the dialog box being displayed on login somewhere else? or just completey removed it? We want to keep this functionality

Reply:
I can move it back